src/modules/app/app.module.ts

```diff
@@ -21,6 +21,7 @@ import { SeedService } from '../seed/seed.service';
     TypeOrmModule.forRoot({
       type: 'postgres',
       url: process.env.DATABASE_URL,
+      schema: process.env.DATABASE_SCHEMA || 'wallet',
```

Comment:
@Kpoke i added the env variable `DATABASE_SCHEMA` here which is defined in the workflow yml file. however the github action is still not working🥲